### PR TITLE
Updates to allow Draco mesh decoding to be viewable (in ThreeJS and BabylonJS) and navigable (#89).

### PR DESCRIPTION
**Pull request by najadojo**

### CODE REVIEW

**Comment by najadojo** [comment]
#89

**Comment by emackey** [comment]
Awesome, thanks!  I'll try to get this reviewed soon.

File: src/gltfPreviewDocumentContentProvider.ts [hunk 1]
```diff
@@ -124,6 +125,7 @@ export class GltfPreviewDocumentContentProvider implements TextDocumentContentPr
 
         // Note that with the file: protocol, we must manually specify the UTF-8 charset.
         const content = this._mainHtml.replace('{assets}',
+            `<script type="text/x-draco-decoder" src="${this.getFilePath('engines/Draco/draco_decoder.js')}"></script>` +
```
Comment (emackey) [inline src/gltfPreviewDocumentContentProvider.ts]: I'd like to replace this with a new string in the existing map.  I'll push a change here.

**Comment by emackey** [comment]
Just pushed a Cesium update to `master` here, so all 3 preview engines support Draco now.

**Pull request merged**